config/config.go

```diff
@@ -0,0 +1,37 @@
+package config
+
+import (
+	"fmt"
+
+	"github.com/spf13/viper"
+)
+
+type Config struct {
+	DBUser     string
+	DBPassword string
+	DBName     string
+	SSLMode    string
+	DBHost     string
+	DBPort     string
+}
+
+func LoadConfig(configFilePath string) (Config, error) {
```

Comment:
you can remove `confgPath` parameter from this method, and just set viper.SetConfigFile(".env") in config ile